### ISSUE TITLE
Model kept in memory despite settings

### ISSUE THREAD

**Issue by andy8992** [n4ze3m/page-assist#192]
I have keep alive set to 0 and I regularly have vram kept in memory. I can easily tell because stable diffusion has far less memory during rendering. This seems to be a new issue. I have now moved to my other llm program to generate prompts during rendering.

**Comment by n4ze3m** [comment]
Yes, I will look into it. Are you using the latest version of Ollama?

**Comment by andy8992** [comment]
I believe so. I notice an update the other day and updated.

**Comment by andy8992** [comment]
Do you have any leads information about this issue? It's still present, I set the time to 1s, 1m or 0 and it still has no effect. The ollama server process still uses several gb of my vram. I can't have that. This extension would be very useful to me if it weren't for that issue.

**Comment by n4ze3m** [comment]
Hey, can you check if Page Assist is causing the issue for you? Try adding a few minutes to the load time, like 10 minutes, from the chat model settings. 
Then, send a message and type this command in the terminal: `ollama ps`. This will show the time until the model is fully loaded.

![image](https://github.com/user-attachments/assets/7ea26156-be4e-45d1-a71e-d2532f311c7a)

![image](https://github.com/user-attachments/assets/12a2b8f8-81b6-4f7a-a4be-c3d2f5948d77)

**Comment by andy8992** [comment]
I change it, and it always says 4 minutes (presumably just not changing from 5)

**Comment by n4ze3m** [comment]
If you configure it here, it will only work for the current chat session. If you start a new chat, the current session will be cleared and reset to default value.

![image](https://github.com/user-attachments/assets/9bd0a2c4-a020-4687-a026-61a26cfce118)


To avoid that, you can set a permanent value here, which will be reused in new chat sessions. This may solve the issue. Let me know if it works for you


https://github.com/user-attachments/assets/dd12711a-aa94-4cbb-8c72-cb7f4c124d74

**Comment by andy8992** [comment]
it doesn't seem to matter, both ways it's staying at 5m 

in task manager, the vram is in ollama_llama_server.exe if that's relevant heh

I can try to reinstall extension, though I have a few prompts that I'd need to re-setup lol

**Comment by andy8992** [comment]
Okay, I reinstalled and it worked for a short time. It seems changing models broke it again. Now it still only does 5 minutes no matter what.
